rebar.config

```diff
@@ -1,5 +1,6 @@
 {erl_opts, [debug_info]}.
 {deps, [
          {nova, "0.9.2"},
```

Comment:
I am thinking about if we only should have Nova here so we always use the latest?

Reply:
Now it should be fine - using the latest nova version.